src/main/java/edu/cornell/kfs/concur/batch/xmlObjects/PdpFeedAccountingEntry.java

```diff
@@ -124,5 +124,13 @@ public KualiDecimal getAmount() {
     public void setAmount(KualiDecimal value) {
         this.amount = value;
     }
+    
+    public String getDebugInfo() {
```

Comment:
I sit corrected, yes it should be toString, I will change this, and a few other places I have used this convention